pkgs/misc/emulators/wine/winetricks.nix

```diff
@@ -11,7 +11,7 @@ stdenv.mkDerivation rec {
   # coreutils is for sha1sum
   pathAdd = lib.concatMapStringsSep ":" (x: x + "/bin")
     (lib.filter (x: x != null)
```

Comment:
I added the filter so I could remove the wine dependency in protontricks, it probably isn't necessary anymore with this change.

Reply:
Good point, have changed to `lib.makeBinPath`.